src/main/java/com/pnu/dev/radioserviceapi/dto/form/NewScheduleItemForm.java

```diff
@@ -0,0 +1,24 @@
+package com.pnu.dev.radioserviceapi.dto.form;
+
+import lombok.Data;
+
+import javax.validation.constraints.NotEmpty;
+
+@Data
+public class NewScheduleItemForm {
+
+    @NotEmpty
+    private String dayOfWeekUrlValue;
+
+    @NotEmpty
+    private String programId;
+
+    private String comment;
+
+    @NotEmpty
+    private String startTime;
+
+    @NotEmpty
+    private String endTime;
```

Comment:
Is it possible to validate if endTime is later than startTime here?

Reply:
Two reasons why it's in separate component:
1. It's better to keep cross field validations in separate component.
2. We anyway need to check whether time period is free or not, so we cannot get rid of `TimeRangeChecker`.